README.md

```diff
@@ -180,3 +180,28 @@ https://www.npmjs.com/package/webpack-bundle-analyzer
 ```
 npx webpack-bundle-analyzer stats.json
 ```
+
+
+## i18n message extraction
+
+To update the translation file, the following needs to be done:
+
+1. Extract the messages from this package:
+
+```
+npm run i18n
+```
+
+2. Copy the widgets.pot file to plone.app.locales
+
+Assuming you are doing this from buildout.coredev:
+```
+cp widgets.pot ../src/plone.app.locales/plone/app/locales/locales/
```

Comment:
I expect the current directory to be `coredev-checkout/src/mockup`. Then the command should be this:

```
cp widgets.pot ../plone.app.locales/plone/app/locales/locales/
```